src/ert/storage/local_experiment.py

```diff
@@ -335,6 +335,43 @@ def response_key_to_response_type(self) -> Dict[str, str]:
         mapping = {}
         for config in self.response_configuration.values():
             for key in config.keys:
+                if key == "*":
+                    continue
+
                 mapping[key] = config.response_type
 
         return mapping
+
+    def update_response_keys(
```

Comment:
Suggest prefacing this with `_` as it should not be used externally.

Reply:
Could you also write a short doc-string explaining why we need to do this? Was not obvious to me before I got it explained.

Reply:
https://github.com/equinor/ert/issues/9059